securitycenter/management_api/event_threat_detection/event_threat_detection_custom_module_test.go

```diff
@@ -54,14 +56,67 @@ func TestMain(m *testing.M) {
 	os.Exit(code)
 }
 
-// extractCustomModuleID extracts the custom module ID from the full name
-func extractCustomModuleID(customModuleFullName string) string {
-	trimmedFullName := strings.TrimSpace(customModuleFullName)
-	parts := strings.Split(trimmedFullName, "/")
-	if len(parts) > 0 {
-		return parts[len(parts)-1]
+func cleanupExistingCustomModules(orgID string) error {
```

Comment:
issue!: Remove this. We sometimes have tests running in parallel--this could potentially delete a resource used by another test.